docs/elevate/ELevating-CentOS6-to-CentOS7.md

```diff
@@ -0,0 +1,165 @@
+  ###### Expected output:
+  ```sh
+  CentOS release 6.10 (Final)
+  ```
+
+* Install the *elevate-release* package that contains the configuration file with CentOS 6 upgrade repositories:
+  ```sh
+  yum install https://repo.almalinux.org/elevate/elevate-release-latest-el6.noarch.rpm
+  ```
+
+* Install the `redhat-upgrade-tool` and `preupgrade-assistant-contents` packages that are used to perform the migration:
+  ```sh
+  yum install redhat-upgrade-tool preupgrade-assistant-contents
+  ```
+
+### Migrate the system
+
+* Run preupgrade assistant as a preugrade check of the system. It will generate the report file with possible problems and risks. No rpm packages will be installed at this phase. 
+  ```sh
+  preupg
+  ```
+
+* The generated **result.html** file is located in the */root/preupgrade/* directory. Inspect it carefully and consider running suitable commands/performing steps to resolve these issues. 
+  
+  **Check the list of risks:**
+  ```sh
+  preupg --riskcheck --verbose
+  ```
+  **An example of how to check the results file using the `lynx` command:**
+  ```sh
+  lynx /root/preupgrade/result.html
+  ```
+
+* Import CentOS 7 GPG key:
+  ```sh
+  rpm --import https://vault.centos.org/7.2.1511/os/x86_64/RPM-GPG-KEY-CentOS-7  
+  ```
+
+* Start an upgrade:
+  ```sh
+   redhat-upgrade-tool --network=7 --cleanup-post --instrepo=http://vault.centos.org/7.2.1511/os/x86_64/
+  ```
+  * The `--cleanup-post` option will remove the CentOS 6 packages remaining after the migration.
+  * The `--network` option sets the system's release version to use online repositories.
+  * You can add the `--force` option to force the migration despite the risks. 
+    :exclamation: Consider this option carefully, and remember to have a system backup. 
+
+* When the migration process is over, reboot the system.
+* A new entry in GRUB called `System Upgrade (redhat-upgrade-tool)` will appear. The system will be automatically booted into it. See how the update process goes in the console. 
+  ::: tip
+  The system will reboot twice during the upgrade.
+  :::
+* After reboot, login to the system. Verify that the current OS is the one you need. 
+  ```sh
+  cat /etc/centos-release
+  ```
+  ###### Expected output:
+  ```sh
+  CentOS Linux release 7.2.1511 (Core) 
+  ```
+  
+### Clean up
+
+* Check packages left from the previous OS version, consider removing or updating them manually.
+  ```sh
+  rpm -qa | grep el6
+  ```
+* Check if any CentOS 6 repositories are left and disable them.
+* Consider removing the CentOS 6 upgrade repository package:
+  ```sh
+  yum remove elevate-release
+  ```
+  
+## Upgrade CenOS 7.2.1511 to CentOS 7.9.2009
+
+* As your *CenOS-Base.repo* file on CentOS 6 was earlier customized to contain CentOS Vault repositories, it wasn't updated during the migration process to contain CentOS 7 repositories. Instead, the new *CentOS-Base.repo.rpmnew* file was created. Run the following commands to replace the *.repo* file with the new one:
```

Comment:
```suggestion
Because the *CentOS-Base.repo* file was customized earlier to contain CentOS Vault repositories, it wasn't updated during the upgrade process to contain CentOS 7 repositories. Instead, the new *CentOS-Base.repo.rpmnew* file would have been created. You can confirm that with these commands:

   ```sh
ls -la /etc/yum.repos.d/CentOS-Base.repo.rpmnew
   ```
```